src/components/Leaderboard.tsx

```diff
@@ -1,65 +1,9 @@
 import React from "react"
-import styled from "@emotion/styled"
 import Emoji from "./OldEmoji"
```

Comment:
This one is deprecated. Lets use `Emoji`. https://github.com/ethereum/ethereum-org-website/blob/dev/docs/chakra-migration-guide.md#update-dependencies